adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Platform/PlatformInformationBase.cs

```diff
@@ -42,10 +42,10 @@ public override string GetAssemblyFileVersionAttribute()
             return assemblyFileVersion != null ? assemblyFileVersion.Version : "internal";
         }
 
-        public async override Task<bool> IsUserLocalAsync(RequestContext requestContext)
-        {
-            return await Task.Factory.StartNew(() => false).ConfigureAwait(false);
-        }
+        //public async override Task<bool> IsUserLocalAsync(RequestContext requestContext)
+        //{
+        //    return await Task.Factory.StartNew(() => false).ConfigureAwait(false);
+        //}
```

Comment:
?

Reply:
Dead code, removed (that method is platform specific, so it goes in PlatformProxy)

---
In reply to: [221416098](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1284#discussion_r221416098) [](ancestors = 221416098)